### PR TITLE
feat: MES- 646 App Landing Page

### PR DESCRIPTION
**Pull request by kaynaran**
### Notes:
- Responsiveness is a bit of a pain – we might need another step of styling adjustments for screen widths below ~500px

### How it's looking:
https://github.com/futuresuper/fs-website-2021/assets/125414123/404900b0-ecee-489b-af39-00b8442c2e7a

### CODE REVIEW

File: src/components/page/App/AppCarousel.svelte [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    scrollPosition = 0;
+  };
+
+  $: if (innerWidth > 900) {
+    cardWidth = 500;
+    resetCarousel();
+  } else {
+    cardWidth = 400;
+    resetCarousel();
+  }
```
Comment (kaynaran) [inline src/components/page/App/AppCarousel.svelte:72]: Changes `cardWidth` with window resizing. Feels a bit hacky, but it works 🤷‍♀️

**Comment by andrewsel** [comment]
Nice work getting this far 🙌 

Still a bit more work to be done on the responsiveness:

* Check on a wide screen as quite a few of the components should be inside a container - I'm pretty sure there is a global class called `container` so if you just add that it should sort you out

image

* For small screens, I usually try and cater down to as low as about 380px and there are a few things to work on at that width:

image

image

image

File: src/components/page/App/AppCarousel.svelte (hunk 1 above)
Comment (andrewsel) [reply src/components/page/App/AppCarousel.svelte:72]: Nothing wrong with doing it this way 👍

File: src/pages/app.astro [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
 ---
 
 <style lang="scss">
   @use "../styles/" as *;
 </style>
 
-<BaseLayout
-  title="Get the Future Super app"
-  description="Future Super has a mobile app for Apple and Android devices"
-  image="https://res.cloudinary.com/future-super/image/upload/v1666674820/app-preview.png"
```
Comment (andrewsel) [inline src/pages/app.astro:13]: I think you need to keep these otherwise the 'preview' won't show up properly in social, google, slack etc

**Review by andrewsel — CHANGES_REQUESTED** [review]
See minor code comments plus above a few things to fix on responsiveness. Almost there! 🙌

**Comment by kaynaran** [comment]
Went back and tried to reflect your suggestions, @andrewsel! For the container issue though, Jean specifically didn't want a hard cut off for some items e.g. hero pic, so I had to selectively apply it. Hope things are looking good 🤞

**Review by andrewsel — APPROVED** [review]
Looks great! 😍

**Pull request merged**